vnext/Microsoft.ReactNative/Modules/AccessibilityInfoModule.cpp

```diff
@@ -42,15 +42,29 @@ void AccessibilityInfo::setAccessibilityFocus(double /*reactTag*/) noexcept {
 void AccessibilityInfo::announceForAccessibility(std::string announcement) noexcept {
   m_context.UIDispatcher().Post([context = m_context, announcement = std::move(announcement)] {
     // Windows requires a specific element to announce from. Unfortunately the react-native API does not provide a tag
-    // So we need to add a temporary control to raise the notification event from.
+    // So we need to find something to raise the notification event from.
+    xaml::UIElement element{nullptr};
 
-    auto textBlock = xaml::Controls::TextBlock();
+    if (react::uwp::Is19H1OrHigher()) {
+      // XamlRoot added in 19H1
+      if (winrt::Microsoft::ReactNative::XamlUIService::GetXamlRoot(context.Properties().Handle())) {
+        if (auto xamlroot =
+                winrt::Microsoft::ReactNative::XamlUIService::GetAccessibleXamlRoot(context.Properties().Handle())) {
+          element = xamlroot.Content();
+        } else {
+          return;
```

Comment:
this would early return out in a UWP app that didn't set the accessible root (but did set xamlroot), which seems wrong

Reply:
Got it! Thought a XAML root was only set for XAML Islands cases, I'll adjust the logic now.